src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidationsTest.java

```diff
@@ -258,6 +258,62 @@ void noCaseIdTest() {
         );
     }
 
+    @Test
+    void caseIdTest() {
```

Comment:
I realise these method names are following a convention started for this class, but maybe this PR would be an opportunity to fix the convention. Ideally, method names should provide short information on:
- what's being tested (unless the class specifies it)
- conditions
- expected outcome

Reply:
@rkondratowicz `[x_]should_y_when_z()` is pretty much what we've got everywhere else. What case does `xTest()` cover?

Reply:
`caseIdTest` is not making that much sense as in what is expected in this test